spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseExpireSnapshotsSparkAction.java

```diff
@@ -88,7 +95,7 @@ public void accept(String file) {
     }
   };
 
-  private final Set<Long> expiredSnapshotIds = Sets.newHashSet();
+  private final Set<Long> snapshotIdsToExpire = Sets.newHashSet();
```

Comment:
I think this is more correct, but I don't think it's worth making this change larger. I would just use a `removedSnapshotIds` variable later to avoid the name conflict.

Reply:
OK, reverted